src/main/java/org/opentripplanner/routing/api/request/RoutingRequest.java

```diff
@@ -731,20 +731,23 @@
     public ItineraryFilterParameters itineraryFilters = ItineraryFilterParameters.createDefault();
 
     /**
-     * The numbers of days before the search date to consider when filtering trips for this search.
-     * This is set to 1 to account for trips starting yesterday and crossing midnight so that they
-     * can be boarded today. If there are trips that last multiple days, this will need to be
-     * increased.
+     * The numbers of days before the search date time to consider when filtering trips for this search.
+     * This is set to 1 to account for trips finishing before midnight on the day before the search
+     * day.
+     *
+     * If you use an arriveBy search and want to return trips starting multiple days before your search
+     * date time, increase this number.
      */
-    public int additionalSearchDaysBeforeToday = 1;
+    public int additionalDaysBeforeSearchTime = 1;
 
     /**
-     * The number of days after the search date to consider when filtering trips for this search.
-     * This is set to 1 to account for searches today having a search window that crosses midnight
-     * and would also need to board trips starting tomorrow. If a search window that lasts more than
-     * a day is used, this will need to be increased.
+     * The number of days after the search date time to consider when filtering trips for this search.
+     * This is set to 1 to account for searches that start after midnight of the search date.
+     *
+     * If you want to return trips spanning multiple days or ones which are multiple days in the future,
+     * increase this number.
      */
-    public int additionalSearchDaysAfterToday = 2;
+    public int additionalDaysAfterSearchTime = 1;
```

Comment:
Maybe we should have only one parameter `additionalDaysToSearch`. It strikes me that this is what we want:
 - `departAfter  =>  [0, additionalDaysToSearch]`
 - `arriveBy =>  [additionalDaysToSearch, 0]`
  
After we changed the patternIndex to be on ALL running dates, we only need to expand the search period to one side. 

Also we could consider using a period `maxTripSearchPeriod` (from EDT to LAT) instead. Then compute the `additionalDaysToSearch`:
```
 additionalDaysToSearch = Duration.between(dateTime, dateTime.plus(maxTripSearchPeriod)).toDays()
```
Example: For a deployment where the estimated longest trip takes 6 hours, and a depart-after search start at 12:00 we can search just one day. If the search start at 21:00 we search today and tomorrow.